card-mod-delayed_job/config/initializers/delayed_job_config.rb

```diff
@@ -1,6 +1,6 @@
-return unless Cardio.config.active_job.queue_adapter == :delayed_job
+return unless Card.config.active_job.queue_adapter == :delayed_job
```

Comment:
here it has to be Cardio.  Card is not loaded yet.

Reply:
oh, maybe it is for now. Sorry.  I want this to be loaded earlier but that's not an option yet...